src/Command/CodeCheckerCommand.php

```diff
@@ -39,6 +39,7 @@ protected function configure(): void
             ->setAliases(['codechecker'])
             ->setDescription('Run Moodle CodeSniffer standard on a plugin')
             ->addOption('standard', 's', InputOption::VALUE_REQUIRED, 'The name or path of the coding standard to use', 'moodle')
+            ->addOption('exclude', null, InputOption::VALUE_REQUIRED, 'Comma separated list of sniff codes to exclude from checking', '')
```

Comment:
This is the only detail I'd argue something, maybe to keep it aligned with the `--standard` option that has the short `-s` equivalent... maybe I'd add `-e` for `--exclude`.

Reply:
I thought about it, but there is currently an `-e` flag for the standalone `phpcs` command (the one you run from php, not from moodle-plugin-ci) that means something else: "[Explain a standard by showing the sniffs it includes](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Usage#getting-help-from-the-command-line)".

So I thought it could be misleading to have it here (an even potentially create issues in the future if we decide to implement that option as well).

But if you still think we should add it, I'm fine with it. Wouldn't be too much of an issue.

Reply:
Okay, done!